gecoscc-installer.sh

```diff
@@ -317,10 +327,12 @@ USER)
 
 PRINTERS)
     echo "LOADING PRINTERS CATALOG"
+    source /opt/rh/python27/enable
```

Comment:
Oops! One of my comments got lost (making the next ones really confusing)

I wrote something like "this repetitive 'source /opt/rh/python27/enable ' should be moved at the begining of the script. Better if existance of Python 2.7 in the server is checked.

 
Template is for another issue (that one about the 'sed' command)